README.md

```diff
@@ -1,8 +1,161 @@
 ![GigaSOM.jl](https://prince.lcsb.uni.lu/GigaSOM.jl/img/logo-GigaSOM.jl.png?maxAge=0)
 
-GigaSOM.jl <br> Huge-scale, high-performance flow cytometry clustering
-===================================================================
+# GigaSOM.jl <br> Huge-scale, high-performance flow cytometry clustering
+
 
 | **Coverage** | **[ARTENOLIS](http://opencobra.github.io/artenolis)** |
 |:------------:|:--------------------------:|
-| [![coverage status](http://codecov.io/github/LCSB-BioCore/GigaSOM.jl/coverage.svg?branch=master)](http://codecov.io/github/LCSB-BioCore/GigaSOM.jl?branch=master) | [![linux](https://prince.lcsb.uni.lu/jenkins/job/GigaSOM.jl-branches-auto-linux/badge/icon)](https://prince.lcsb.uni.lu/jenkins/job/GigaSOM.jl-branches-auto-linux/) |
\ No newline at end of file
+| [![coverage status](http://codecov.io/github/LCSB-BioCore/GigaSOM.jl/coverage.svg?branch=master)](http://codecov.io/github/LCSB-BioCore/GigaSOM.jl?branch=master) | [![linux](https://prince.lcsb.uni.lu/jenkins/job/GigaSOM.jl-branches-auto-linux/badge/icon)](https://prince.lcsb.uni.lu/jenkins/job/GigaSOM.jl-branches-auto-linux/) |
+
+# How to get started
+
+## Prerequisites
+
+- Use Linux (Debian and centOS), macOS X, or Windows 10 as your operating system. `GigaSOM` has been tested on these systems.
+- In order to use `GigaSOM`, you need to install Julia 1.0 or higher. You can find the download and installation instructions for Julia [here](https://julialang.org/downloads/).
+
+Once `julia` has been installed, follow [these instructions](https://docs.julialang.org/en/v1/manual/getting-started/) to get started.
+
+:bulb: If you are a complete beginner with Julia, you can follow an extensive tutorial [here]. Primarily, you should familiarize yourself with the [Julia package manager](https://julialang.github.io/Pkg.jl/v1/getting-started/).
```

Comment:
The actual link to `[here]` is still missing.

Reply:
Actually I would avoid any "here" links, they are not great in general ;-)

Reply:
Point taken 👍